docs/configuring.md

```diff
@@ -893,6 +893,18 @@ to know available options.
 
 The CLI option can be passed several times to add several patterns.
 
+Each pattern can be templated using Jinja.
+
+!!! example
+
+    Templating `exclude` patterns using `_operation` allows to have files
+    that are rendered once during `copy`, but are never updated:
+
+    ```yaml
+    _exclude:
+        - "{%- if _operation == 'update' -%}src/*_example.py{%- endif %}"
```

Comment:
Since there's no need to strip whitespaces, let's keep it simple and avoid confusing readers as to why we're showing this expression with the marker:

```suggestion
        - "{% if _operation == 'update' %}src/*_example.py{% endif %}"
```